Submodule public.sdk contains modified content
diff --git a/public.sdk/source/common/threadchecker_mac.mm b/public.sdk/source/common/threadchecker_mac.mm
index 7ccdc35..a735084 100644
--- a/public.sdk/source/common/threadchecker_mac.mm
+++ b/public.sdk/source/common/threadchecker_mac.mm
@@ -19,6 +19,7 @@
 #if SMTG_OS_MACOS
 
 #include <pthread.h>
+#include <exception>
 #include <Foundation/Foundation.h>
 
 //------------------------------------------------------------------------
